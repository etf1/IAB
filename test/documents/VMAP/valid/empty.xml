<!-- Most basic valid VMAP document -->
<vmap:VMAP xmlns:vmap="http://www.iab.net/vmap-1.0" version="1.0">
</vmap:VMAP>
