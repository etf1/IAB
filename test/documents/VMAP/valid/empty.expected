{
  "version": "1.0",
  "breaks": []
}
